ios/Classes/MapboxMapController.swift

```diff
@@ -71,53 +72,59 @@ class MapboxMapController: NSObject, FlutterPlatformView, MGLMapViewDelegate, Ma
             result(FlutterMethodNotImplemented)
         }
     }
-    
+
     private func updateMyLocationEnabled() {
-        //TODO
+      mapView.showsUserLocation = self.myLocationEnabled
+      mapView.userTrackingMode = .followWithHeading
     }
-    
+
     private func getCamera() -> MGLMapCamera? {
         return trackCameraPosition ? mapView.camera : nil
     }
-    
+
     /*
      *  MGLMapViewDelegate
      */
     func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
         isMapReady = true
         updateMyLocationEnabled()
-        
+
         if let initialTilt = initialTilt {
             let camera = mapView.camera
             camera.pitch = initialTilt
             mapView.setCamera(camera, animated: false)
         }
-        
+
         mapReadyResult?(nil)
     }
-    
+
     func mapView(_ mapView: MGLMapView, shouldChangeFrom oldCamera: MGLMapCamera, to newCamera: MGLMapCamera) -> Bool {
         guard let bbox = cameraTargetBounds else { return true }
         // Get the current camera to restore it after.
         let currentCamera = mapView.camera
-        
+
         // From the new camera obtain the center to test if it’s inside the boundaries.
         let newCameraCenter = newCamera.centerCoordinate
-        
+
         // Set the map’s visible bounds to newCamera.
         mapView.camera = newCamera
         let newVisibleCoordinates = mapView.visibleCoordinateBounds
-        
+
         // Revert the camera.
         mapView.camera = currentCamera
-        
+
         // Test if the newCameraCenter and newVisibleCoordinates are inside bbox.
         let inside = MGLCoordinateInCoordinateBounds(newCameraCenter, bbox)
         let intersects = MGLCoordinateInCoordinateBounds(newVisibleCoordinates.ne, bbox) && MGLCoordinateInCoordinateBounds(newVisibleCoordinates.sw, bbox)
-        
+
         return inside && intersects
     }
-    
+
+    func mapView(_ mapView: MGLMapView, didChange mode:
+    MGLUserTrackingMode, animated: Bool) {
+      channel.invokeMethod("map#onCameraTrackingChanged", arguments: ["mode": mode.rawValue])
```

Comment:
Given that iOS don't have `onCameraTrackingDismissed`, to maintain compatibility with Android when mode is equals to `MGLUserTrackingModeNone` invoke this method `map#onCameraTrackingDismissed`